mapiproxy/servers/default/emsmdb/emsmdbp_object.c

```diff
@@ -2735,17 +2739,43 @@ static int emsmdbp_object_get_properties_mapistore_root(TALLOC_CTX *mem_ctx, str
 	uint32_t			contextID;
         uint32_t                        *obj_count;
 	uint8_t				*has_subobj;
-	/* time_t				unix_time; */
-	/* NTTIME				nt_time; */
-	/* struct FILETIME			*ft; */
+	uint32_t			*folder_flags;
+	uint64_t			*fid;
 
 	contextID = emsmdbp_get_contextID(object);
 
 	folder = (struct emsmdbp_object_folder *) object->object.folder;
         for (i = 0; i < properties->cValues; i++) {
-                if (properties->aulPropTag[i] == PR_CONTENT_COUNT) {
+		if (properties->aulPropTag[i] == PidTagFolderFlags) {
+			/* Folder flags / yet undocumented */
+			folder_flags = talloc_zero(data_pointers, uint32_t);
+			*folder_flags = 0x5;
```

Comment:
I have specifically added "yet undocumented" for this reason. The property is not documented in Microsoft specifications and I am waiting for more information about it for Microsoft.